go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/distatus/battery v0.10.1-0.20200722221337-7e1bf2bbb15c
 	github.com/go-ole/go-ole v1.2.4 // indirect
 	github.com/gookit/color v1.3.1
+	github.com/kevinburke/go-bindata v3.22.0+incompatible // indirect
```

Comment:
I'm wondering, isn't go mod complaining about this as it isn't referenced anywhere?

Reply:
@TravisTX maybe we can split the commits here and add this addition, and the use of `go generate` in the contributing guide. We can add a line in `main.go` to tell `go generate` to use `go-bindata`:

```golang
//go:generate go-bindata -o init.go init/
```

This needs to be on the first line, as indicated in the [documentation](https://blog.golang.org/generate).

Reply:
@TravisTX you know what, I'll facilitate this change. We need to make sure all the build scripts are aligned (same for homebrew).

Reply:
I'm new to Go, so I understood about half of what you just said 😄.  So, it's probably good that you're handling that. I look forward to seeing your change, so I can learn from it!